Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
             targets: ["FaceLiveness"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/aws-amplify/amplify-swift", from: "2.15.4")
+        .package(url: "https://github.com/aws-amplify/amplify-swift", exact: "2.27.2")
```

Comment:
Do we want to do `exact`? that requries an release of this package if amplify-swift is updated.

Reply:
discussed offline, this is expected behavior

Reply:
yup, this is intended - due to future work in `amplify-swift` repository for liveness